templates/home/landingpage.html.twig

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}login
+{% endblock %}
+
+{% block stylesheets %}
+	{{ parent() }}
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap" rel="stylesheet">
+{% endblock %}
+
+{% block body %}
+	<div class="landing-page">
+		<div class="logo-container">
+			<img class="logo" src="{{ asset('build/images/Motorboxcf.png') }}">
+		</div>
+
+		<section class="landing-button-container">
+			<div>
+				<button class="landing-button">
+					S'inscrire
+				</button>
+			</div>
+			<div>
+				<button class="landing-button">
+					Se connecter
+				</button>
+			</div>
```

Comment:
the div looks useless here... and your buttons will end up as links, so prefer <a> tags